// react-hanger/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var useArray_1 = require("./useArray");
Object.defineProperty(exports, "useArray", { enumerable: true, get: function () { return useArray_1.useArray; } });
var useBoolean_1 = require("./useBoolean");
Object.defineProperty(exports, "useBoolean", { enumerable: true, get: function () { return useBoolean_1.useBoolean; } });
var useInput_1 = require("./useInput");
Object.defineProperty(exports, "useInput", { enumerable: true, get: function () { return useInput_1.useInput; } });
var useLogger_1 = require("./useLogger");
Object.defineProperty(exports, "useLogger", { enumerable: true, get: function () { return useLogger_1.useLogger; } });
var useMap_1 = require("./useMap");
Object.defineProperty(exports, "useMap", { enumerable: true, get: function () { return useMap_1.useMap; } });
var useNumber_1 = require("./useNumber");
Object.defineProperty(exports, "useNumber", { enumerable: true, get: function () { return useNumber_1.useNumber; } });
var usePrevious_1 = require("./usePrevious");
Object.defineProperty(exports, "usePrevious", { enumerable: true, get: function () { return usePrevious_1.usePrevious; } });
var useSetState_1 = require("./useSetState");
Object.defineProperty(exports, "useSetState", { enumerable: true, get: function () { return useSetState_1.useSetState; } });
var useStateful_1 = require("./useStateful");
Object.defineProperty(exports, "useStateful", { enumerable: true, get: function () { return useStateful_1.useStateful; } });
